docs/configuration.md

```diff
@@ -2,7 +2,7 @@
 
 The operation of the syzkaller `syz-manager` process is governed by a configuration file, passed at
 invocation time with the `-config` option.  This configuration can be based on the
-[example](syz-manager/config/testdata/qemu.cfg); the file is in JSON format with the
+[example](../syz-manager/mgrconfig/testdata/qemu.cfg); the file is in JSON format with the
```

Comment:
I think it will be more resilient to future changes if we give links as `/syz-manager/mgrconfig/testdata/qemu.cfg`.
Same for the other one.

Reply:
Yes, you're right. I'll use the absolute link and commit soon.